css/admin.css

```diff
@@ -460,8 +460,48 @@ body #post-head-content p.wp-clearfix:nth-child(1) {
 		font-size: 14px;
 		white-space: nowrap;
 	}
+
+	.oo-google-recaptcha-key {
```

Comment:
media stylings should be a the end of css file to overwrite the styling on different devices. Please move the whole media block to the end and I'm sure you can remove the !important tag